src/plugins/controls/public/options_list/components/options_list_popover_sorting_button.tsx

```diff
@@ -85,17 +85,16 @@ export const OptionsListPopoverSortingButton = ({
     },
     [optionsList.dispatch]
   );
-
   const SortButton = () => (
     <EuiButtonEmpty
       size="s"
       color="text"
       iconSide="right"
       iconType="arrowDown"
       disabled={showOnlySelected}
+      className="optionsList__sortButton"
       data-test-subj="optionsListControl__sortingOptionsButton"
       onClick={() => setIsSortingPopoverOpen(!isSortingPopoverOpen)}
-      className="euiFilterGroup" // this gives the button a nice border
```

Comment:
Refer to https://github.com/elastic/kibana/pull/162651/files#r1283499840 for why I removed this.